data/presets/tourism/wilderness_hut.json

```diff
@@ -3,18 +3,23 @@
     "fields": [
         "name",
         "operator",
-        "address",
         "building_area",
+        "access_simple",
         "fee",
         "payment_multi_fee",
         "charge_fee",
         "fireplace"
     ],
     "moreFields": [
+        "address",
+        "capacity",
+        "drinking_water",
```

Comment:
this field is currently labeled _Drinkable_. It could be slightly confusing to see that come up in the context of a hut. It's probably necessary to create a dedicated version of the `drinking_water` field with a more generic label (e.g. `Drinking Water Available`).

Reply:
//edit: the `tourism=camp_site` preset could also benefit from such a field

Reply:
I've now implemented this in 2e3825c